lib/client-assets.php

```diff
@@ -244,7 +244,9 @@ function gutenberg_register_packages_scripts( $scripts ) {
  * `build/` location.
  *
  * @since 6.7.0
-
+ *
+ * @global array $editor_styles.
```

Comment:
```suggestion
 * @global array $editor_styles
```

Let's make it the same as the text in the core.

https://github.com/WordPress/wordpress-develop/blob/0acfd3a72c66c7544b3ef9e183ee1af3958ae3c3/src/wp-includes/block-editor.php#L776
https://github.com/WordPress/wordpress-develop/blob/0acfd3a72c66c7544b3ef9e183ee1af3958ae3c3/src/wp-includes/theme.php#L1604